lib/improver/utilities/solar.py

```diff
@@ -278,6 +278,7 @@ def process(self, cube):
         for i, dtval in enumerate(dtvalues):
             mask_cube = daynight_mask[i]
             day_of_year = (dtval - dt.datetime(dtval.year, 1, 1)).days
+            dtval = dtval + dt.timedelta(seconds=dtval.second)
```

Comment:
This seems to push the timestamp forward by e.g. 58 seconds, as an example the timestamp is updated from e.g. 2018-09-12 05:42:58.999986 to 2018-09-12 05:43:56.999986. Is this what we want? I can see that this might be desired if the aim is for the terminator to be accurate to the nearest minute. If so, then it would be good to add a bit more information to the docstring to say that the aim is for the terminator to be accurate to the nearest minute.

Reply:
I've added an explanation to the doc string.